pypesto/store/save_to_hdf5.py

```diff
@@ -59,7 +59,9 @@ def write(self, problem, overwrite: bool = False):
             for problem_attr in attrs_to_save:
                 value = getattr(problem, problem_attr)
                 if isinstance(value, (list, np.ndarray)):
-                    write_array(problem_grp, problem_attr, value)
```

Comment:
in case of an empty array the first condition in the if statement of `write_array` will be true, so all empty ones will be saved as int arrays. This messes up shape for example.